### PR TITLE
Add SLES12 SP3 autoyast profiles of s390x

### PR DESCRIPTION
**Pull request by dehai**
add those profiles for s390x zVM upgrade test cases
those files include default and all patterns,registered and fully update

- Related ticket: https://progress.opensuse.org/issues/33481
- Verification run: 
   SLES12 SP3, default install, proxy SCC -- http://bartok.arch.suse.de/tests/239
   SLES12 SP3, all patterns, Packages media  -- http://bartok.arch.suse.de/tests/237
   SLES12 SP3 + SDK + HA + Geo, all patterns, proxy SCC -- http://bartok.arch.suse.de/tests/233
   SLES12 SP3 + SDK + HA + Geo, default install, Packages media -- http://bartok.arch.suse.de/tests/236
   SLES12 SP3, all addons, default install, proxy SCC -- http://bartok.arch.suse.de/tests/232
   SLES12 SP3, all addons, all patterns, Packages media -- http://bartok.arch.suse.de/tests/238

### CODE REVIEW

**Comment by rwx788** [comment]
@okurz there are quite some issues we were not able to fix on Friday. I'll check if we can make it work on zVM at least, on zKVM it won't work.

File: data/autoyast_sle12/autoyast_sles12sp3_default_reg_full_s390x.xml [hunk 1]
```diff
@@ -132,7 +132,7 @@
   </software>
   <suse_register>
     <do_registration config:type="boolean">true</do_registration>
-    <reg_code>25af9a31773574ab</reg_code>
+    <reg_code>{{SCC_REGCODE}}</reg_code>
```
Comment (rwx788) [inline data/autoyast_sle12/autoyast_sles12sp3_default_reg_full_s390x.xml]: Oh, seriously, we had a regcode in the profile on github =)

**Comment by Soulofdestiny** [comment]
but for the profiles i don't see any issues - +1

**Comment by dehai** [comment]
@rwx788 I think no, those profiles are just for s390x,  and just for SLES12 SP3 installation, the verification run is passed with installation part, the upgrade parts , I think its my worker problem or something else,  there is a test case running at OSD already, here is https://openqa.suse.de/tests/1583783, I need add those cases to run at OSD to check the upgrade parts. thanks!

**Comment by rwx788** [comment]
@dehai alright, let's merge then and see how it works there. Meanwhile me and @Soulofdestiny work on some improvements for ay installations on s390x. Let's see how it goes.

**Pull request merged**